changelogs/8.15.asciidoc

```diff
@@ -1,7 +1,19 @@
 [[apm-release-notes-8.15]]
 == APM version 8.15
+* <<apm-release-notes-8.15.1>>
 * <<apm-release-notes-8.15.0>>
 
+[float]
+[[apm-release-notes-8.15.1]]
+=== APM version 8.15.1
+
+https://github.com/elastic/apm-server/compare/v8.15.0\...v8.15.1[View commits]
+
+[float]
+==== Bug fixes
+
+- Fixes Elasticsearch apm-data plugin to fallback to ILM policy for datastreams created with APM integration and not explicitly migrated to Datastream Lifecycle {pull}13918[13918]
```

Comment:
Should we include your PR's note in the changelog as well?

`NOTE: Any indices created when APM is on version 8.15.0 and datastream created before 8.15.0 i.e. with ILM, will remain Unmanaged even after this fix. To fix them, we would need to explicitly update them OR use the PUT API on datastream to set DSL.`

Reply:
I have linked the doc to the known-issues page which has more details.